packages/java-parser/test/samples-spec.js

```diff
@@ -9,6 +9,7 @@ const javaParser = require("../src/index");
 
 describe("The Java Parser", () => {
   createSampleSpecs("java-design-patterns");
+  createSampleSpecs("guava");
```

Comment:
A failing test is a start but not too useful.

Could we modify the "createSamplesSpec" to only fail **after**
N files have failed? that could be a naive (and imperfect) way to "fix" a current state and avoid regressions.

We could even create a new createFailingSampleSpec("guava", 101)
API specifically for that scenario that will parse all the sample's file in a **single** spec.

If we can do that than we can merge this PR and incrementally work on reducing the number of failures in multiple small PRs.